src/main/resources/META-INF/rewrite/jakarta-ee-9.yml

```diff
@@ -373,12 +403,6 @@ recipeList:
       newGroupId: jakarta.faces
       newArtifactId: jakarta.faces-api
       newVersion: latest.release
-  - org.openrewrite.java.dependencies.ChangeDependency:
-      oldGroupId: org.glassfish
-      oldArtifactId: javax.faces
-      newGroupId: jakarta.faces
-      newArtifactId: jakarta.faces-api
-      newVersion: latest.release
```

Comment:
What's the background to this removal? Since I do see a moved artifact: https://mvnrepository.com/artifact/org.glassfish/javax.faces

Reply:
As far as i've seen, the dependencies with org.glassfish groupId are implementations. This artifact got moved to org.glassfish:jakarta.faces, which consists of the JSF RI "Mojarra". I think moving an implementation to just the specs is unsafe. 

However, I'll just change this to a migration towards org.glassfish:jakarta.faces instead of jakarta.faces:jakarta.faces-api

Reply:
Fyi there some similar jakarta API implementation artifacts by glassfish. Are you interested in a PR adding similar migrations?

Reply:
Thanks for the added context ! Let's indeed remove this one then and hold off on any glassfish implementation specific migrations. If there's demand in the future we could always add those to a separate recipeList.